src/docs/content/observability/observability-scenario/scenario.md

```diff
@@ -0,0 +1,74 @@
+---
+title: "Scenario"
+date: 2020-03-10T13:29:23-06:00
+draft: false
+tags:
+  - observability in action
+---
+
+### Pre-Requisites
+
+- Deploy [Mushop]({{< ref "pre-requisites/setup.md" >}})
+- Setup Oracle Cloud Infrastructure (OCI) Logging [OCI Logging]({{< ref "pre-requisites/oci-logging.md" >}})
+- Setup Oracle Cloud Infrastructure (OCI) Notifications [OCI Notifications]({{< ref "pre-requisites/oci-notifications.md" >}})
+- Setup Oracle Cloud Infrastructure (OCI) Service Connector Hub [OCI Service Connector Hub]({{< ref "pre-requisites/oci-service-connector.md" >}})
+
+### Payment Failures
+
+Navigate to MuShop application at [http://localhost:8000](http://localhost:8000) if using ```kubectl port-forward``` as discussed under [Mushop]({{< ref "pre-requisites/setup.md" >}}) and add items to cart to exceed the amount greater than $105.
+The request will be denied with HTTP 406 "Request Not Acceptable"
+
+Click on "place order" 4-5 times just to create some additional failure log data.
+
+### Observe
+
+Navigate to OCI Console ``Monitoring -> Metrics Explorer``
```

Comment:
this only has double backtick, where others use triple. My suggestion is to always use single backtick for these.